test/e2e/framework/framework.go

```diff
@@ -37,9 +37,7 @@ const (
 	PluginSubCommand     = "tanzu %s"
 
 	// Central repository
-	CentralRepositoryFeatureFlag         = "features.global.central-repository"
-	CentralRepositoryPreReleaseRepoImage = "TANZU_CLI_PRE_RELEASE_REPO_IMAGE"
```

Comment:
I also removed this constant which was not used.  We should not duplicate the string value here, but use the global value defined in the CLI.